node/node.go

```diff
@@ -1044,11 +1044,16 @@ func NewNode(config *cfg.Config,
 	}
 
 	// Initialize ValidatorConnExecutor
-	validatorConnExecutor := dashquorum.NewValidatorConnExecutor(
+	validatorConnExecutor, err := dashquorum.NewValidatorConnExecutor(
 		nodeInfo.ID(),
 		eventBus,
 		sw,
-		logger.With("module", "ValidatorConnExecutor"))
+		dashquorum.WithLogger(logger),
+		dashquorum.WithValidatorsSet(state.Validators),
```

Comment:
These validators are not necessarily correct. ABCI app, in response to InitChain, can provide a set of validators that should be used. See https://github.com/dashevo/tenderdash/blob/cf7cff1a4803535292e4ef396cd8f11d9b9decf3/consensus/replay.go#L415-L416 

I would suggest just to ensure that in this case, proper event is published to the event bus.

Reply:
the suspicion was checked, the validators were taken from the init-chain